app/views/govuk_publishing_components/components/_chart.html.erb

```diff
@@ -92,14 +92,15 @@
       } %>
     <% end %>
 
-    <div class="gem-c-chart__chart" id="<%= chart_id %>">
+    <% aria_attributes = { hidden: true } if minimal %>
```

Comment:
I think if we have `aria-hidden` i don't think it can be keyboard focusable as well, as the component guide is flagging this as an issue. Happy for you to pick between either having focus or having `aria-hidden`.

Reply:
Argh, good spot, thanks. Think I've fixed this now - since the chart is just a link, I've included some logic to simply not render the visually hidden text (it's the thing that contained a link).